glide.lock

```diff
@@ -126,7 +126,7 @@ imports:
 - name: github.com/ghodss/yaml
   version: 0ca9ea5df5451ffdf184b4428c902747c2c11cd7
 - name: github.com/go-kit/kit
-  version: fbab14bd4c487151b206c2a41ecfbb015e32c1e0
+  version: da68e7640663097af890b9bdd76da9cb816ebb82
```

Comment:
weird to see glide.lock changes without glide.yaml, wanna just revert to old versions?